swagger.json

```diff
@@ -7427,6 +7434,13 @@
             "Generated",
             "UserDefined"
           ]
+        },
+        "voxelSampleRate": {
+          "description": "the amount of sub voxels per voxel in each direction (x,y,z)",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 10
```

Comment:
I'm not sure how this file got included in this repo. I'm not a fan of having it in here. While it (theoretically) provides the swagger the client was created from, it gives the wrong impression that this is the canonical specification when in fact simulation-api-specification contains the real spec. I'd prefer we not include it in this repo.

Reply:
I'm open to negotiation on this.

Reply:
Understand, you were the one that added it originally so though I would follow.  I will remove it.

Reply:
It figures...